elements/pfe-tabs/src/pfe-tabs.js

```diff
@@ -231,14 +231,16 @@ class PfeTabs extends PFElement {
     const tabs = this._allTabs();
     const tab = tabs[index];
 
-    if (!tab) {
+    if (tabs.length > 0 && !tab) {
```

Comment:
This checks now if any number of tabs exist and that specific tab does not vs. no tabs existing at all which is a separate issue.